composer.json

```diff
@@ -18,8 +18,9 @@
         "php": ">=5.4.0 <8.0",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "phpunit/phpunit": ">4.8.20 <6.0",
-        "phpunit/php-code-coverage": ">=2.2.4 <5.0",
+        "phpunit/phpunit": ">4.8.20 <6.1",
```

Comment:
I wasn't sure about this one, I read this comment on the issue https://github.com/Codeception/Codeception/issues/3995#issuecomment-284078436 and I tried to use a small range for the versions. If that's not a problem for you all, I can use a wider set of versions then.